src/openklant/config/tests/test_admin.py

```diff
@@ -182,3 +182,25 @@ def test_existing_klantcontacten_with_invalid_kanalen_raises_error(self):
         self.assertFalse(config.enabled)
         self.assertIsNone(config.service)
         self.assertEqual(config.kanalen_tabel_code, "")
+
+    def test_status_check_returns_http_status_and_items(self):
+        response = self.app.get(self.url)
+        form = response.forms["referentielijstenconfig_form"]
+
+        form["enabled"] = True
+        form["service"] = self.service.pk
+        form["kanalen_tabel_code"] = "KANAAL"
+
+        form.submit()
+
+        config = ReferentielijstenConfig.get_solo()
+
+        try:
+            result = config.status_check()
```

Comment:
This is a general point: tests that only test internal methods like this are not enough to test functionality. Currently this `status_check` method is not used anywhere except in this test, so if I were to merge this now, this status check functionality would not be part of the admin at all. This is also why I think it's important to not just rely on unittests while developing something, but to also manually check if it works in the admin

There should at least be one test on the admin view to verify that this method is actually called there, that way we know that the functionality is actually used